src/components/file/FileCard.js

```diff
@@ -40,25 +40,35 @@ function FileCard({
   const branch = (file && file.branch) ? file.branch : repository.default_branch;
 
   return (
-    <Card>
+    <Card
+      data-test="component-fileCard"
+    >
       <CardHeader
         avatar={<Avatar src={avatarUrl} />}
         title={<strong>{file && file.path}</strong>}
         subheader={repository.full_name + '/' + branch}
+        data-test="cardHeader"
       />
       <CardContent>
         <Paper>
           <BlockEditable
             preview={preview}
             markdown={markdown}
+            // NEED HELP
             onEdit={setMarkdown}
-            editable={!!authentication}
+            editable={isAuthenticated}
+            data-test="blockEditable"
           />
```

Comment:
@ancientTexts-net 
how can I simulate the onEdit event with just `BlockEditable.simulate('onEdit', mock Event);` doesn't work